CppSource/Include/xsapi/social_manager_c.h

```diff
@@ -141,7 +141,7 @@ typedef struct XSAPI_SOCIAL_EVENT
     XSAPI_XBOX_LIVE_USER* user;
     XSAPI_SOCIAL_EVENT_TYPE eventType;
     XSAPI_XBOX_USER_ID_CONTAINER** usersAffected;
-    uint32_t numOfUsersAffected;
+    size_t usersAffectedSize;
```

Comment:
nit: usersAffectedCount, size could imply allocation size in bytes

Reply:
here and below

---
In reply to: [154734283](https://github.com/Microsoft/xbox-live-unity-plugin/pull/160#discussion_r154734283) [](ancestors = 154734283)